docs/book/src/developer/providers/v1.2-to-v1.3.md

```diff
@@ -41,6 +41,7 @@ in Cluster API are kept in sync with the versions used by `sigs.k8s.io/controlle
 - A new timeout `nodeVolumeDetachTimeout` has been introduced that defines how long the controller will spend on waiting for all volumes to be detached.
 The default value is 0, meaning that the volume can be detached without any time limitations.
 - A new annotation `machine.cluster.x-k8s.io/exclude-wait-for-node-volume-detach` has been introduced that allows explicitly skip the waiting for node volume detaching.
+- A new annotation `"cluster.x-k8s.io/replicas-managed-by": "external-autoscaler"` has been introduced to indicate that replica enforcement is delegated to an external autoscaler (not managed by Cluster API). When this annotation is present the `MachinePool` controller will no longer predict the "direction" of scaling when the `MachinePool` replica count is different from the observed `Ready` replicas, but will instead simply log a phase status update that the `MachinePool` is actively scaling.
```

Comment:
Sounds good to me. WDTY about linking to the more extensive documentation in machine-pool.md?

(then you can also drop some details if you want)

Reply:
good, idea, done